volatility3/framework/symbols/windows/pdbconv.py

```diff
@@ -941,8 +941,8 @@ def determine_extended_value(
         return name_str, value, excess
 
 
-class PdbRetreiver:
-    def retreive_pdb(
+class PdbRetriever:
```

Comment:
Same as above (and actually, these ones are more important, the development ones I'd be ok with just breaking, since they shouldn't be used directly by anyone).